amaranth-stubs/lib/wiring.pyi

```diff
@@ -56,7 +56,7 @@ Out = Flow.Out
 In = Flow.In
 
 @final
-class Member:
+class Member(Generic[_T_Signature]):
```

Comment:
This weakens typing: anywhere just `Member` is used, it defaults to `Member[Unknown]`, and therefore `member.signature()` is no longer typed. And `Member` is returned from `__getitem__` on `SignatureMembers`, so typing this precisely is not possible.

Does it make sense to just use `AbstractSignature` instead of `_T_Signature`? Does this break anything?

By the way, the reason `AbstractSignature` exists in the first place is because `FlippedSignature` is not a `Signature`.

Reply:
Yes, It makes the same sense. I thought it using Generic would improve typing on `.signature` in case Member is made from signature, but I see this is not a case. I will revert.